### DISCUSSION TITLE
Getting the value for the Rotation around Z from and Open3D Transformation Matrix

### DISCUSSION THREAD

**Discussion by Dalecn109** [isl-org/Open3D#3327] [Q&A] [answered]
So I have a transformation matrix which I have got in Open3D after comparing Point Clouds but I want to find out from that matrix the exact value for rotation around the Z axis so I can use it else where. What would be the mathematical formula to work this out?

**Comment by theNded** [comment]
Search `matrix to euler angles` on Google and there are a plethora of references, e.g. https://stackoverflow.com/questions/15022630/how-to-calculate-the-angle-from-rotation-matrix

**Reply by germanros1987** [reply, marked as answer]
@Dalecn109 please, consider marking this question as answered.
